pkg/daemon/drain.go

```diff
@@ -39,12 +39,16 @@ func (dn *Daemon) cordonOrUncordonNode(desired bool) error {
 	}
 	var lastErr error
 	if err := wait.ExponentialBackoff(backoff, func() (bool, error) {
+		// Log has been added to ensure that MCO is correctly performing cordon/uncordon.
+		// This should help us with debugging bugs like https://bugzilla.redhat.com/show_bug.cgi?id=2022387
+		glog.Infof("Initiating %s on node (currently unschedulable: %t)", verb, dn.node.Spec.Unschedulable)
```

Comment:
Can we add an event here for the initiating cordon/uncordon like we have in : 
https://github.com/openshift/machine-config-operator/blob/76d6155c1dab8050db4d911c32d3e7113faedbfd/pkg/daemon/drain.go#L110

Reply:
Not sure, adding event here would mean during retry multiple events will be generated. Also, this is less error prone area so capturing these additional log in daemon should be sufficient for debugging.